src/services/CityConfigService.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+
+import type {Config} from '../Config'
+import format from 'string-format'
+import CityConfig from '../cities/CityConfig'
+import cityConfigs from '../cities/cityConfigs'
+
+export default class CityConfigService {
+  config: Config
+
+  constructor (config: Config) {
+    this.config = config
+  }
+
+  getFilledCityConfigs (): Array<CityConfig> {
+    const baseUrl = `${this.config.host}:${this.config.port}`
+    // Object.values() only returns an Array<mixed>, see https://github.com/facebook/flow/issues/2221 for reference.
+    // Therefore we disable flow type checking for this line.
+    // $FlowFixMe
+    const cityConfigsArray: Array<CityConfig> = Object.values(cityConfigs)
+    cityConfigsArray.forEach(cityConfig => {
+      cityConfig.logo = format(cityConfig.logo, baseUrl)
+      cityConfig.favicon = format(cityConfig.favicon, baseUrl)
+    })
```

Comment:
You're modifying the original cityConfig objects here :warning: :warning: 
Don't do that

Reply:
I think at the moment making the city configs be returned by a function is the easiest way. Another way would be to deep copy the object manually, but that's a bit hacky IMO.

Reply:
Hmm... I'm not sure how you'd implement the deep copy thing, but we shouldn't lose the prototype of the model...

Reply:
Done.